scripts/main.js

```diff
@@ -302,4 +302,53 @@ class Cart {
     clear() {
         this.storage.clear();
     }
+}
+
+/*Here starts engaging with product's add buttons.
+* I synchronized buttons with Cart, and on click
+* i'm parsing html product into object and create
+* instance of ProductModel with its data.
+* Then i'm adding this instance to localStorage via Cart.*/
+const cart = new Cart();
+const bagCounters = document.getElementById("bag");
+const addBtns = document.getElementsByClassName("product__addBtn");
+
+for (let i = 0; i < addBtns.length; i++) {
+    addBtns[i].addEventListener("click", function () {
```

Comment:
Do listener function not na anonym function. Define handler function as saperate and independent.